api/service.go

```diff
@@ -99,6 +100,13 @@ func registerServices(ctx context.Context, cfg Config, grpcServer *grpc.Server,
 		return err
 	}
 
+	// account api
```

Comment:
Does it mean we do not need the above `config api` registration right?

Reply:
we have two apis since `GET /config` endpoint is not removed and its a separate API from now on.

Reply:
why do we need the `GET` endpoint exposed if we read from the file always?

Reply:
I felt its nicer to be able to see the config using an API, since we had the API thought of not leaving it as it is.